platform/openide.nodes/src/org/openide/cookies/InstanceCookie.java

```diff
@@ -31,7 +31,7 @@
  *
  * @author Jaroslav Tulach
  */
-public interface InstanceCookie/*<T>*/ extends Node.Cookie {
+public interface InstanceCookie<T> extends Node.Cookie {
```

Comment:
This is an API change.

Reply:
Reverted back to original.

Reply:
I've reverted this back to the original state..